From 395c60c2bc00f9f2685d662c006736e2fa562808 Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Mon, 24 Sep 2012 16:39:19 +0300
Subject: [PATCH] libnl: add lnl-genl to default configuration

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 libnl-2.0.pc.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libnl-2.0.pc.in b/libnl-2.0.pc.in
index e44f0fb..0bca566 100644
--- a/libnl-2.0.pc.in
+++ b/libnl-2.0.pc.in
@@ -6,5 +6,5 @@ includedir=@includedir@
 Name: libnl
 Description: Convenience library for netlink sockets
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lnl
+Libs: -L${libdir} -lnl -lnl-genl
 Cflags: -I${includedir}
-- 
1.7.9.5

